### ISSUE TITLE
No quantized layers in this solution

### ISSUE THREAD

**Issue by afonso-sousa** [pyjhzwh/LQ-net-pytorch#3]
Hello. First of all, congratulations on your work. I am not completely familiarized with this work yet, but analysing the original implementation I see that they use quantized versions of conv layers and quantize activations on the network architecture. Can you please give me some insights on how is this done in your version of the work?

**Comment by pyjhzwh** [comment]
Right. My implementation does not include the quantization of activation layers. I just quantize the weights of convolutional layers.
